### ISSUE TITLE
Cowardice enchantment is bugged

### ISSUE THREAD

**Issue by Krill18** [Thelnfamous1/Dungeons-Gear#103]
On the latest update, 3.0.17, having the Cowardice enchantment on any equipped armor piece halves the player's max HP.

![2021-07-05_17 03 20](https://user-images.githubusercontent.com/86990587/124525358-e2ddc000-ddb3-11eb-9299-95c3acb09ad7.png)

![2021-07-05_17 03 27](https://user-images.githubusercontent.com/86990587/124525389-06086f80-ddb4-11eb-805c-978e43a263eb.png)

**Comment by Raverbury** [comment]
Can confirm this happened to me as well. I think it got mixed up with the Reckless enchantment since I lost 6 hearts (60% of my max health)

**Comment by Raverbury** [comment]
Tested it again a regular zombie, using the Spear from this mod. No other modifiers and no critical hits, all attacks are made after the attack indicator has filled all the way up. Without Cowardice III a hit does 3 hearts, with it while at full health does 7.5 hearts and while not at full health does 5.5 hearts. Looks to me like both Cowardice and Reckless are present

**Comment by Thelnfamous1** [comment]
Fixed for next update

**Comment by AcidArmy** [comment]
Problem: Cowardice enchantment removes 60% of maximum health when on equipped armor.

Forge 36.2.0
Minecraft 1.16.5
MCP 20210115.111550
4 mods loaded: Minecraft, Forge, Dungeons Gear, Spice of Life: Carrot Edition

Steps to replicate:

1. Create a new creative world with default settings.
2. Apply Cowardice to leather pants
3. Go into survival mode
4. Equip the leather pants

I suspected it was Spice of Life: Carrot Edition originally, but I tested with and without it installed and at various maximum HP values. Results do not differ.
